funding/manager.go

```diff
@@ -315,6 +315,15 @@ func (m *Manager) deriveFundingShim(ourOrder order.Order,
 		selfChanBalance = matchedOrder.Order.(*order.Bid).SelfChanBalance
 	}
 
+	// If either order requires script enforcement, our thaw height needs to
+	// be an absolute height instead of a relative one.
+	switch {
+	case ourOrder.Details().ChannelType == order.ChannelTypeScriptEnforced:
+		fallthrough
+	case matchedOrder.Order.Details().ChannelType == order.ChannelTypeScriptEnforced:
+		thawHeight += batchHeightHint
```

Comment:
Shouldn't this instead add the height hint fuzz factor? Otherwise we end up doubling (?) what we think the final lease expiration height should be.

Reply:
The `thawHeight` will continue to be relative, unless a script enforced channel is to be opened, in which case we convert it to an absolute height as required by adding the proposed batch height.